pairing/circl_bls12381/adapter.go

```diff
@@ -0,0 +1,48 @@
+package circl_bls12381
+
+import (
+	"go.dedis.ch/kyber/v3"
+)
+
+// SuiteBLS12381 is an adapter that implements the suites.Suite interface so that
+// bls12381 can be used as a common suite to generate key pairs for instance but
+// still preserves the properties of the pairing (e.g. the Pair function).
+//
+// It's important to note that the Point function will generate a point
+// compatible with public keys only (group G2) where the signature must be
```

Comment:
is this not the reverse of the standard case? It seems that most bls libs put pub keys on G1 and sigs on G2. In drand we do the opposite.
It seems circl supports both